jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
oh, ok. Is it because the `appPath` in the `codeSign` step is pointing to that dir? So basically the codeSign plugin will make sure the signed app will be always in that dir?